babel.config.js

```diff
@@ -0,0 +1,9 @@
+// We use Babel for our tests in scripts/.
+module.exports = {
+  env: {
+    test: {
+      presets: ['@babel/preset-typescript'],
+      plugins: ['@babel/plugin-transform-modules-commonjs'],
+    },
+  },
+};
```

Comment:
babel is used for the create-package tests that I think we want to use to create new packages